pkgs/development/python-modules/requests-kerberos/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage, requests, pykerberos, mock }:
+
+buildPythonPackage rec {
+    pname = "requests-kerberos";
+    version = "0.12.0";
+    name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.